libexec/tgenv-install

```diff
@@ -20,7 +20,7 @@ if [[ "${version_requested}" =~ ^latest\:.*$ ]]; then
   version="${version_requested%%\:*}"
   regex="${version_requested##*\:}"
 elif [[ "${version_requested}" =~ ^latest$ ]]; then
-  version="${version_requested}"
+  version="$(bash tgenv-get-latest-release)"
```

Comment:
I wonder how the original tgenv did it, compared to your implementation?
because I remember we removed it, not remember if by accident or not.

Reply:
I think this because we want to minimize the calls to `"https://api.github.com` because rate limit. 

@amit-alkobi-env0 is the only option in env0 to get the latest when you configure in the file `latest`?

Reply:
> I think this because we want to minimize the calls to "https://api.github.com because rate limit.

True, see [this PR](https://github.com/env0/tgenv/pull/10)
> is the only option in env0 to get the latest when you configure in the file latest?

Obviously we can think of many other ways to get the latest version, for example, having a procedure/ chore that updates list_all_versions_offline regularly.
But I don't think this will result with a substantially more robust solution as tgenv is also dependent on a GitHub API call to download the package, which I assume is robust enough as we currently use it.

Note that as oppose to [the way it is done in original repo](https://github.com/cunymatthieu/tgenv/blob/master/libexec/tgenv-install#L31), the `repos/gruntwork-io/terragrunt/releases/latest` is used, which doesn't have an arbitrary response size, so there is a good reason to believe that it won't get rate-limited.

What is your concern with the suggested solution?

Reply:
If the only option to get the latest option is write it to tgenv file (in their repo). I am ok with it.

Reply:
> Note that as oppose to the way it is done in original repo, the repos/gruntwork-io/terragrunt/releases/latest is used, which doesn't have an arbitrary response size, so there is a good reason to believe that it won't get rate-limited.

This API indeed looks better, also you call it only when the user asked for "latest",
so it is different from the original repo when `tgenv-list-remote` is called always.

So Im ok with that, we can test it by time.